.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
 install:
   - ./travis-tool.sh r_binary_install knitr
   - ./travis-tool.sh r_binary_install testthat
+  - ./travis-tool.sh r_install pryr
```

Comment:
Is it possible to use lazyeval instead of pryr? All promise related stuff is moving towards lazyeval.